chardonnay/genesis.json

```diff
@@ -626,6 +658,25 @@
         "next_channel_sequence": "0"
       }
     },
+    "interchainaccounts": {
+      "controller_genesis_state": {
```

Comment:
Not sure how the submodule structure and naming is meant to work for these modules, but we're not running the controller, only the host. In the v6 upgrade we manually set the controller params to an empty value:

https://github.com/PeggyJV/sommelier/blob/bd0a767f599e6665e8a059e5db0ca6492eec07ec/app/upgrades/v6/upgrades.go#L29

Not 100% on how this is reflected in a genesis state but we definitely don't want to set `controller_enabled` to true.